test_hello_worlds/hello.ts

```diff
@@ -0,0 +1 @@
+console.log("hello world");
```

Comment:
This file contains at least one console log. Please remove any present. - additions.length